brand/identity/index.md

```diff
@@ -41,43 +57,69 @@ layout: app
+			<p class="p-2 bg-c-g100 t-center">Do not use the old logos</p>
+		</div>
+		<div class="col-4">
+			<img class="border t-c-g100 m-bottom-0" src="{{site.baseurl}}/assets/images/logo/dh-ldn-2.png" alt="Dreamhost logo - do not change color">
+			<p class="p-2 bg-c-g100 t-center">Do not change the logo colors</p>
+		</div>
+		<div class="col-4">
+			<img class="border t-c-g100 m-bottom-0" src="{{site.baseurl}}/assets/images/logo/dh-ldn-3.png" alt="Dreamhost logo - do not change type">
+			<p class="p-2 bg-c-g100 t-center">Do not change the logo type</p>
+		</div>
+	</div>
 </div>
-<div class="u-clearfix">
-	<div class="g-1_3"><img class="border t-c-g100 m-bottom-0" src="{{site.baseurl}}/assets/images/logo/dh-ldn-4.png" alt="Dreamhost logo - do not rotate"><p class="p-2 bg-c-g100 t-center">Do not rotate the logo</p></div>
-	<div class="g-1_3"><img class="border t-c-g100 m-bottom-0" src="{{site.baseurl}}/assets/images/logo/dh-ldn-5.png" alt="Dreamhost logo - do not distort"><p class="p-2 bg-c-g100 t-center">Do not distort the logo</p></div>
-	<div class="g-1_3"><img class="border t-c-g100 m-bottom-0" src="{{site.baseurl}}/assets/images/logo/dh-ldn-6.png" alt="Dreamhost logo - do not add effects"><p class="p-2 bg-c-g100 t-center">Do not add effects to the logo</p></div>
+<div class="container p-x-0">
+	<div class="row">
+		<div class="col-4">
+			<img class="border t-c-g100 m-bottom-0" src="{{site.baseurl}}/assets/images/logo/dh-ldn-4.png" alt="Dreamhost logo - do not rotate">
+			<p class="p-2 bg-c-g100 t-center">Do not rotate the logo</p>
+		</div>
+		<div class="col-4">
+			<img class="border t-c-g100 m-bottom-0" src="{{site.baseurl}}/assets/images/logo/dh-ldn-5.png" alt="Dreamhost logo - do not distort">
+			<p class="p-2 bg-c-g100 t-center">Do not distort the logo</p>
+		</div>
+		<div class="col-4">
+			<img class="border t-c-g100 m-bottom-0" src="{{site.baseurl}}/assets/images/logo/dh-ldn-6.png" alt="Dreamhost logo - do not add effects">
+			<p class="p-2 bg-c-g100 t-center">Do not add effects to the logo</p>
+		</div>
+	</div>
 </div>
 
 <hr>
 
 <h2 class="t-5 m-bottom-4">Typography</h2>
 <p class="t-4 m-bottom-12">Typography is the visual component of our written communications. It's what makes reading easier, establishing content hierarchy and making the text easier to digest. Consistent use of type makes for a successful customer experience.</p>
 
-<div class="u-clearfix">
-	<div class="g-1_3">
-		<h3 class="m-bottom-3">Ubuntu</h3>
-		<p>Ubuntu is a versatile display font that we use for headlines on our website, blog and printed collateral.</p>
-		<p>Note that we only use the ‘Medium’ weight of the Ubuntu font family. If you feel the need to use a thinner or heavier weight, please contact the brand team before doing so.</p>
-	</div>
-	<div class="g-2_3">
-		<img src="{{site.baseurl}}/assets/images/type/type-ubuntu.png" alt="Ubuntu" />
+<div class="container p-x-0">
+	<div class="row">
```

Comment:
Can we make this row responsive so that the font image breaks to the next line on small screens? Probably `md` breakpoint.

Reply:
I added col-md-6 to make them stack evenly on mobile